locales/en-US/messages.properties

```diff
@@ -15,10 +15,13 @@ newProjectBtn=Start a project from scratch
 contactEmail=Email Us
 contactTwitter=Talk to us on Twitter
 readTheDocs=Read the Docs
-contribute=Contribute on Github
+contribute=Contribute on GitHub
```

Comment:
Correct, but this is mostly a special case. We need to bump the string ID when the meaning has changed, here it’s basically a typo in English only, localizers are supposed to catch those and fix them in their language. For instance, it’s fixed in German https://pontoon.mozilla.org/de/thimble/all-resources/?search=GitHub&string=149094

So, we don’t need to ask everyone to retranslate this string